VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
same with https://github.com/uizaio/api-wrapper-java/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R15
is bester